Conditions/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+
+namespace Conditions
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //2
+
+            double operand1 = 15;
+            double operand2 = 0;
+            double result;
+
+            Console.WriteLine("Enter the sign of the arithmetic operation:");
+
+            string sign = Console.ReadLine();
+
+            switch (sign)
+            {
+                case "+":
+                    result = operand1 + operand2;
+                    Console.WriteLine("operand1 {0} operand2 = {1:#.###}", sign, result);
+                    break;
+                case "-":
+                    result = operand1 - operand2;
+                    Console.WriteLine("operand1 {0} operand2 = {1:#.###}", sign, result);
+                    break;
+                case "*":
+                    result = operand1 + operand2;
+                    Console.WriteLine("operand1 {0} operand2 = {1:#.###}", sign, result);
+                    break;
+                case "/":
+                    if (operand2 == 0)
+                    {
+                        Console.WriteLine("Division by zero!");
+                        break;
```

Comment:
don't do the same action in both if and else scopes, do it after if else instead